src/components/GuardianIdeal/GuardianIdealCarousel.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { useRef } from 'react'
+import { useFlags } from 'launchdarkly-react-client-sdk'
+import Slider from 'react-slick'
+import { Button } from '@trussworks/react-uswds'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import styles from './GuardianIdealCarousel.module.scss'
+import 'slick-carousel/slick/slick.css'
+import 'slick-carousel/slick/slick-theme.css'
+import GuardianIdealItem from './GuardianIdealItem'
+import { WidgetWithSettings } from 'components/Widget/Widget'
+import { IdealListItem, Widget } from 'types'
+import { useModalContext } from 'stores/modalContext'
+
+const CustomEllipse = ({ onClick }: any) => {
+  return (
+    <div className="news-carousel-container">
+      <button
+        onClick={onClick}
+        type="button"
+        title="slide"
+        className={styles.newsCarouselEllipse}
+      />
+    </div>
+  )
+}
+
+const GuardianIdealCarousel = ({
+  ideals,
+  widget,
+}: {
+  ideals: IdealListItem[]
+  widget: Widget
+}) => {
+  const { updateModalId, updateModalText, modalRef, updateWidget } =
+    useModalContext()
+
+  const sliderRef = useRef<Slider>(null)
+  const flags = useFlags()
+
+  const settings = {
+    dots: true,
+    fade: true,
+    accessibility: true,
+    adaptiveHeight: true,
+    infinite: true,
+    speed: 500,
+    slidesToShow: 1,
+    slidesToScroll: 1,
+    arrows: false,
+    dotsClass: `slick-dots ${styles.dots}`,
+    appendDots: (dots: React.ReactNode) => {
+      return (
+        <div>
+          <button
```

Comment:
the carousel arrows need some sort of labeling for screen reader users. This is the issue I am seeing on my end when I look at it in ANDI. 

<img width="1532" alt="image" src="https://user-images.githubusercontent.com/59394696/220719719-4e62cc88-3554-4c4a-909a-8fd7c610505a.png">

some info for remediation can be found here: https://www.ssa.gov/accessibility/andi/help/alerts.html?no_name_generic

if you need to pair or have questions, let me know!

Reply:
@shkeating I tested with ANDI and looks like the alerts went away after I added some `aria-label` attributes, but let me know if you see any other issues!